code/modules/cooking/machinery/cooking_machines/fryer.dm

```diff
@@ -166,7 +152,7 @@
 	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN*3)
 
 	if(!victim || !victim.Adjacent(user))
```

Comment:
You removed the delay so you don't need to check if the victim ran away now.

Reply:
No, it never checks outside of the proc.

Reply:
I meant that we do some victim checking on line 147. It'd be better for the adjacent check to be next to that. I don't see why it needs to have the click cooldown thing inbetween it?